app/views/banners/_form.html.erb

```diff
@@ -21,9 +21,10 @@
       <%= form.label :name %>
       <%= form.text_field :name, required: true %>
     </div>
-    <div class="form-check checkbox-style mb-20">
-      <%= form.check_box :active, class: 'form-check-input' %>
+    <div data-controller="reveal" data-reveal-hidden-class="d-none" class="form-check checkbox-style mb-20">
+      <%= form.check_box :active, data: { action: @org_has_alternate_active_banner && 'click->reveal#toggle' }, class: 'form-check-input' %>
       <%= form.label :active, "Active?", class: 'form-check-label' %>
+      <span data-reveal-target="item" class="text-danger <%= conditionally_add_hidden_class(form.object.active) %>">Warning: This will replace your current active banner</span>
```

Comment:
Would it be better to actually use a Stimulus class here? https://stimulus.hotwired.dev/reference/css-classes

Reply:
Ah, might be! Will be out of town this weekend, but will look into this early next week.

Reply:
I believe we already do make use of a stimulus class the parent `div` element with `<div ... data-reveal-hidden-class="d-none" ...>`, and this pattern is what was given in the `stimulus-reveal-controller` documentation.

I'm still solidifying my mental model for how everything works with stimulus, but I'm not 100% sure of how to make use of a stimulus class for the above, as we're just setting the initial classes of the span which are then toggled by the stimulus controller.

Reply:
Okay no worries, I'm not familiar with the stimulus-reveal-controller and how it would interact with a Stimulus class. Fine as-is.